.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "standard-react"
+  ],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "react/jsx-indent": [ 2, 4 ]
```

Comment:
I went beyond the "eslint:recommended" basics and also included [eslint-config-standard-react](https://github.com/feross/eslint-config-standard-react) which includes [eslint-config-jsx](https://github.com/feross/eslint-config-standard-jsx).

This is where the JSX indent is set to 2 spaces, which is why I need to specify it explicitly here:
- https://github.com/feross/eslint-config-standard-jsx/blob/master/eslintrc.json#L20